mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -79,6 +79,49 @@ func resourceContainerNodePool() *schema.Resource {
 	}
 }
 
+var schemaBlueGreenSettings = &schema.Schema{
+	Type:     schema.TypeList,
+	Optional: true,
+	Computed: true,
```

Comment:
```suggestion
	Computed: true,
	MaxItems: 1,
```
Still propose to add `MaxItems: 1` here, as I believe it's not actually a list or array but a nested object.

Reply:
Looks like a push just includes this change, remove the comments then.